### PR TITLE
wallet_balance.py: Prevent edge cases

### PR DESCRIPTION
**Pull request by stevenroose**
I ran into this edge case when running the test on Elements. I had a 0-value output as change.

### CODE REVIEW

File: test/functional/wallet_balance.py [hunk 1]
```diff
@@ -30,10 +30,15 @@ def create_transactions(node, address, amt, fees):
         ins_total += utxo['amount']
         if ins_total + max(fees) > amt:
```
Comment (MarcoFalke) [inline test/functional/wallet_balance.py]: Shouldn't this also say?
```suggestion
        if ins_total >= amt + max(fees):
```

**Comment by MarcoFalke** [comment]
ACK, ran into the same issue yesterday. Thanks for fixing before I could

File: test/functional/wallet_balance.py (hunk 1 above)
Comment (stevenroose) [reply test/functional/wallet_balance.py]: ah yeah I first had that but then rebased on master and noticed that someone already added that `max(fees)` check. I will edit it to be `>=` instead.

Comment (stevenroose) [reply test/functional/wallet_balance.py]: changed

**Comment by promag** [comment]
utACK bb41e63.

**Pull request merged**